src/postgres/interface/static.vendor.h

```diff
@@ -210,7 +210,7 @@ typedef PageHeaderData *PageHeader;
 // ---------------------------------------------------------------------------------------------------------------------------------
 /*
  * PageIsNew
- *		returns true if page has not been initialized (by PageInit)
+ *		returns true iff page has not been initialized (by PageInit)
```

Comment:
Is this truly "If and only if" now? I really don't like the "iff" but I'm assuming this is how it is now written in PostgreSQL code and we have not control over it?

Reply:
Yeah, it's always been like that in the PG code. I'm not sure how it got be be "if" in our code. I must have reflexively spell-checked it. I think I'll commit this one separately, though.